filter_linux.go

```diff
@@ -552,7 +552,7 @@ func AdjustSize(sz uint, mpu uint, linklayer int) uint {
 	}
 }
 
-func CalcRtable(rate *nl.TcRateSpec, rtab [256]uint32, cellLog int, mtu uint32, linklayer int) int {
+func CalcRtable(rate *nl.TcRateSpec, rtab *[256]uint32, cellLog int, mtu uint32, linklayer int) int {
```

Comment:
[]uint32

Reply:
I think this should be consistent with https://github.com/shemminger/iproute2/blob/v2.6.31/tc/q_htb.c#L107